i18n/en.js

```diff
@@ -56,17 +56,17 @@ export default {
     submit: "Submit",
   },
   skillLevels: {
-    1: "I don't know it",
-    2: "I know a bit",
-    3: "I alredy tested/used it a little",
-    4: "I used it a lot",
-    5: "I'm a pro",
+    1: "I looked at what it was",
+    2: "I played with it, I tested it",
```

Comment:
Ces 2 options utilisent un passé simple alors que les suivantes utilisent un composé. Je suggère d'aligner.
```suggestion
    1: "I've looked at what it was",
    2: "I've played with it, I've tested it",
```